usecases/schema/class.go

```diff
@@ -275,10 +284,20 @@ func (h *Handler) setClassDefaults(class *models.Class) {
 	}
 
 	if class.ReplicationConfig == nil {
-		class.ReplicationConfig = &models.ReplicationConfig{Factor: 1}
+		class.ReplicationConfig = &models.ReplicationConfig{Factor: int64(globalCfg.MinimumFactor)}
+	}
+
+	if class.ReplicationConfig.Factor > 0 && class.ReplicationConfig.Factor < int64(globalCfg.MinimumFactor) {
```

Comment:
Would be good to handle the case here when `Factor = 0`.

Reply:
it would be caught because min. is 1 , see test case https://github.com/weaviate/weaviate/pull/5669/files#diff-705ccc396e6acc267c5a336267ad658a662feb411e2c3734aa999044376612d8R1629